### DISCUSSION TITLE
indented code blocks

### DISCUSSION THREAD

**Discussion by gabriele-ct** [mdx-js/mdx#2562] [Q&A] [answered]
If I try to run this md in a md playground

```
## Code blocks

Note again how the actual text starts at 4 columns in (4 characters
from the left side). Here's a code sample:

    # Let me re-iterate ...
    for i in 1 .. 10 \{ do-something(i) }
```

I get the last 2 lines rendered as code block (an old project running mdx v1) was doing the same
<img width="610" alt="Screenshot 2024-11-25 at 15 54 05" src="https://github.com/user-attachments/assets/a34ea49d-7698-4cd6-b0ef-86c31762bd78">

However if I try to run the same code in the [mdxjs playground](https://mdxjs.com/playground/) I don't see the same code block formatting applied.
<img width="794" alt="Screenshot 2024-11-25 at 15 54 11" src="https://github.com/user-attachments/assets/abf13db6-1bc4-4ef1-a4c8-e94a7e1106c8">

does that mean that mdx v3 doesn't support indented code blocks?

Thank you

**Comment by remcohaszing** [comment, marked as answer]
> does that mean that mdx v3 doesn't support indented code blocks?

That’s correct. This is documented in https://mdxjs.com/docs/what-is-mdx/#markdown.

**Reply by gabriele-ct** [reply]
Thank you!
